docs/guides/Localization and Internationalization.md

```diff
@@ -48,6 +48,7 @@ The upcoming road or ramp destination is named according to the local or nationa
 | Hebrew     | ✅              | ✅                             | —                     | ✅
 | Hungarian  | ✅              | —                             | —                     | ✅
 | Italian    | ✅              | ✅                             | ✅                     | ✅
+| Korean    | ✅              | —                             | ✅                     | ✅
```

Comment:
The sample applications currently won’t show anything in Korean, but any application that is localized into Korean will have the fully localized navigation UI.

Spoken instructions are marked as unavailable for now, until we release and deploy a version of the Directions API that incorporates Project-OSRM/osrm-text-instructions#243.